docs/pages/components/checkbox.md

```diff
@@ -0,0 +1,199 @@
+---
```

Comment:
If we're going to move checkboxes to it's own page (👍 ), we should remove the examples from the "Form" page and provide a link to this page in it's place.